schema.sql

```diff
@@ -10,4 +10,15 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL(10,2)
-);
\ No newline at end of file
+);
+
+-- Creating the animals table
+CREATE TABLE animals (
+    id INT,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL(10,2),
+    species VARCHAR(100)
```

Comment:
Hi, thank you for your submission. I have a few comments on your code:

The assignment asked you to use the ALTER TABLE command to add a new column called species with a string type. However, you Created a new TABLE, then added the species column directly to the new CREATE TABLE statement. This does not demonstrate your understanding of how to modify an existing table using SQL commands. Please revise your code to use the ALTER TABLE command as instructed. Please kindly find below an example:

`ALTER TABLE animals ADD species varchar(255);`